code/modules/mob/living/carbon/human/emote_vr.dm

```diff
@@ -130,6 +130,30 @@
 			message = "lets out a wurble."
 			m_type = 2
 			playsound(src, 'sound/voice/wurble.ogg', 50, 1, -1, preference = /datum/client_preference/emote_noises)
+		if("snort")
+			message = "snorts!"
+			m_type = 2
+			playsound(src, 'sound/voice/Generic-Snort.ogg', 50, 0, preference = /datum/client_preference/emote_noises)
+		if("vlaugh")
+			message = "lets out a laugh."
+			m_type = 2
+			playsound(src, 'sound/voice/Generic-laugh.ogg', 50, 0, preference = /datum/client_preference/emote_noises)
+		if("vchuckle")
+			message = "gives a small chuckle."
+			m_type = 2
+			playsound(src, 'sound/voice/Generic-Chuckle.ogg', 50, 0, preference = /datum/client_preference/emote_noises)
+		if("Ccackles")
```

Comment:
Ccackles? Is the extra C for extra sekrit?

Reply:
>-< so much for screets. The C stand for a certian job. Its slightly a longer sounding emote that could be horrible if people found it out and abused it.

Reply:
Lol, no worries @H0lySquirr3l. In that case I'll delete my comments - if you do the same it'll re-hide this.